components/automate-cli/pkg/verifyserver/server/api/v1/stopmockserver.go

```diff
@@ -0,0 +1,90 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) StopMockServer(c *fiber.Ctx) {
+
+	reqBody := new(models.StopMockServerRequestBody)
+	err := c.BodyParser(&reqBody)
+
+	// If request body is invalid
+	if err != nil {
+		errMsg := fiber.Error{
+			Code:    fiber.StatusBadRequest,
+			Message: fmt.Sprintf("Stop mock server request body parsing failed: %v", err.Error()),
+		}
+		h.Logger.Error(fmt.Errorf(errMsg.Message))
```

Comment:
```suggestion
		h.Logger.Error(errMsg.Message)
```
We dont need fmt.Errorf. Please make this changes at all places

Reply:
done